Makefile

```diff
@@ -1,6 +1,21 @@
+GOPROXY ?= https://goproxy.io
+GOOS ?= linux
+GOARCH ?= amd64
```

Comment:
Regarding the default behavior for OS and arch, my bet is, we should set the default as darwin/arm64 (M1 Mac users) instead of Linux/amd64.

Reply:
I don’t know which user scenario is more for mac and linux. Maybe there are more macs for writing code and more linux for running. We can discuss it more.